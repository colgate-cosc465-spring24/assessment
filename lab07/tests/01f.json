{
    "ases" : [
        {
            "number": 1,
            "prefix": "11.0.0.0/8"
        },
        {
            "number": 2,
            "prefix": "12.0.0.0/8"
        },
        {
            "number": 3,
            "prefix": "13.0.0.0/8"
        },
        {
            "number": 4,
            "prefix": "14.0.0.0/8"
        }
    ],
    "relationships" : [
        {
            "customer": 1,
            "provider": 2
        },
        {
            "peerA": 2,
            "peerB": 3
        },
        {
            "provider": 3,
            "customer": 4
        }
    ]
}
