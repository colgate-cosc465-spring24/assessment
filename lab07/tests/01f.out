***** Topology *****
AS 1 (11.0.0.0/8): cust=[]; prov=[2]; peer=[]
AS 2 (12.0.0.0/8): cust=[1]; prov=[]; peer=[3]
AS 3 (13.0.0.0/8): cust=[4]; prov=[]; peer=[2]
AS 4 (14.0.0.0/8): cust=[]; prov=[3]; peer=[]
***** Routes *****
AS 1
	11.0.0.0/8: 
	12.0.0.0/8: 2
	13.0.0.0/8: 2 -> 3
	14.0.0.0/8: 2 -> 3 -> 4
AS 2
	11.0.0.0/8: 1
	12.0.0.0/8: 
	13.0.0.0/8: 3
	14.0.0.0/8: 3 -> 4
AS 3
	11.0.0.0/8: 2 -> 1
	12.0.0.0/8: 2
	13.0.0.0/8: 
	14.0.0.0/8: 4
AS 4
	11.0.0.0/8: 3 -> 2 -> 1
	12.0.0.0/8: 3 -> 2
	13.0.0.0/8: 3
	14.0.0.0/8: 
